client/src/stories/Page.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import { Header } from './Header';
+import './page.css';
+
+export const Page = () => {
+  const [user, setUser] = React.useState();
+
+  return (
+    <article>
+      <Header
+        user={user}
+        onLogin={() => setUser({ name: 'Jane Doe' })}
+        onLogout={() => setUser(undefined)}
```

Comment:
```suggestion
        onLogout={() => setUser(null)}
```
"undefined" is set by the system, null is set by the developer.